WordPressKitTests/BlockEditorSettingsServiceRemoteTests.swift

```diff
@@ -42,23 +49,25 @@ extension BlockEditorSettingsServiceRemoteTests {
             }
             waitExpectation.fulfill()
         }
-        mockRemoteApi.successBlockPassedIn!(mockedResponse, HTTPURLResponse())
 
-        waitForExpectations(timeout: 0.1)
-        validateEditorThemeRequest()
+        wait(for: [waitExpectation], timeout: 0.1)
     }
 
     func testFetchThemeNoGradients() {
-        let waitExpectation = expectation(description: "Theme should be successfully fetched")
-        var mockedResponse = mockedData(withFilename: twentytwentyoneResponseFilename) as! [[String: Any]]
+        stub(condition: isHost("public-api.wordpress.com") && isPath("/wp/v2/sites/1/themes") && containsQueryParams(["status": "active"])) { _ in
+            var mockedResponse = self.mockedData(withFilename: self.twentytwentyoneResponseFilename) as! [[String: Any]]
+
+            // Clear out Gradients
+            var theme = mockedResponse[0]
+            var themeSupport = theme[RemoteEditorTheme.CodingKeys.themeSupport.stringValue] as! [String: Any]
+            themeSupport[RemoteEditorThemeSupport.CodingKeys.gradients.stringValue] = "false"
+            theme[RemoteEditorTheme.CodingKeys.themeSupport.stringValue] = themeSupport
+            mockedResponse[0] = theme
```

Comment:
From the PR description:

> The main motivation of this change is, I can reuse the same unit tests (with minimal code changes) in the upcoming `BlockEditorSettingsServiceRemote` refactor.

Would you also consider extracting this and the other response data manipulations in dedicated methods? It would for sure improve readability. That's useful per se, but at this level I wouldn't make a strong case for it unless there was a case for reuse. If the tests are going to be reused somehow, then the refactor investment to extract some of this code would be useful.